packages/components/src/styles/footnotes/footnotes.stories.ts

```diff
@@ -41,10 +45,6 @@ export default {
   ]
 };
 
-/**
- * Default: This shows sd-footnote in its default state, used with a `ul` element.
```

Comment:
For the 1st sample please use white background

Reply:
Updated